x-pack/plugins/observability_solution/slo/server/assets/ingest_templates/slo_pipeline_template.ts

```diff
@@ -17,6 +17,12 @@ export const getSLOPipelineTemplate = (slo: SLODefinition) => ({
   id: getSLOPipelineId(slo.id, slo.revision),
   description: `Ingest pipeline for SLO rollup data [id: ${slo.id}, revision: ${slo.revision}]`,
   processors: [
+    {
+      set: {
+        field: '_id',
+        value: `${slo.id}-${slo.revision}-{{{_id}}}`,
```

Comment:
```suggestion
        value: `{{{_id}}}-${slo.id}-${slo.revision}`,
```

Reply:
I initially used the suffix approach then the prefix one. I don't know if one is better than the other. Any reason for this?

Reply:
It's probably not a deal but I guess different string at start is better than at end.
In a string comparison, that might matter at nano scale